### ISSUE TITLE
Broken in RubyMine 2016.2

### ISSUE THREAD

**Issue by tinynumbers** [basgren/railways#25]
Running in:
- RubyMine 2016.2
- Build #RM-162.1236.14, built on July 14, 2016
- JRE: 1.8.0_76-release-b216 x86_64
- JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
- Railways plugin version 0.8.5

Attempting to refresh the list of routes results in the following exception:

```
org.jetbrains.plugins.ruby.ruby.lang.psi.expressions.RListOfExpressions.getElement(I)Lorg/jetbrains/plugins/ruby/ruby/lang/psi/RPsiElement;
java.lang.NoSuchMethodError: org.jetbrains.plugins.ruby.ruby.lang.psi.expressions.RListOfExpressions.getElement(I)Lorg/jetbrains/plugins/ruby/ruby/lang/psi/RPsiElement;
	at net.bitpot.railways.utils.RailwaysPsiUtils.findMethodInClassModules(Unknown Source)
	at net.bitpot.railways.utils.RailwaysPsiUtils.findControllerMethod(Unknown Source)
	at net.bitpot.railways.models.RailsActionInfo.update(Unknown Source)
	at net.bitpot.railways.models.routes.SimpleRoute.updateActionStatus(Unknown Source)
	at net.bitpot.railways.utils.RailwaysUtils.updateActionsStatus(Unknown Source)
	at net.bitpot.railways.routesView.RoutesManager.parseRakeRoutesOutput(Unknown Source)
	at net.bitpot.railways.routesView.RoutesManager.access$500(Unknown Source)
	at net.bitpot.railways.routesView.RoutesManager$UpdateRoutesTask.onSuccess(Unknown Source)
	at com.intellij.openapi.progress.impl.CoreProgressManager.finishTask(CoreProgressManager.java:412)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl$1.a(ProgressManagerImpl.java:152)
	at com.intellij.openapi.application.TransactionGuardImpl$6.run(TransactionGuardImpl.java:291)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.a(LaterInvocator.java:326)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:310)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at com.intellij.ide.IdeEventQueue.h(IdeEventQueue.java:863)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:699)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:385)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by basgren** [comment]
Thank you for such a detailed report! 
The issue is fixed in the latest 0.8.6 plugin version. Please check.

**Comment by tinynumbers** [comment]
Excellent, thanks @basgren for the quick turn-around.
